framework/base/ErrorHandler.php

```diff
@@ -100,6 +108,10 @@ public function handleException($exception)
             }
             $this->renderException($exception);
             if (!YII_ENV_TEST) {
+                \Yii::getLogger()->flush(true);
```

Comment:
Overall flushing log makes sense on exception. Any reason why it's done for `if (!YII_ENV_TEST) {`?